Package.swift

```diff
@@ -1392,6 +1398,52 @@ func firestoreWrapperTarget() -> Target {
   )
 }
 
+func firebaseFirestoreCppTarget() -> Target {
```

Comment:
Why an additional target versus modifying the existing Firestore target?

Will pipelines be an optional feature to be configured into the SDK?